bubblesub/ui/main_window.py

```diff
@@ -208,6 +218,30 @@ def _setup_menu(self) -> None:
             HotkeyContext.Global,
         )
 
+    def _restore_widgets_visibility(self) -> None:
+        for (key, data) in self._api.cfg.opt["gui"]["visibility"].items():
+            widget = self.findChild(QtWidgets.QWidget, key)
+
+            if data[1]:
+                widget = widget.parentWidget()
+
+            widget.setEnabled(data[0])
+            widget.setVisible(data[0])
```

Comment:
I'm not sure why do you set the hidden widgets as disabled? You can't interact with them anyway, but it complicates the code and the config

Reply:
It was a copy-paste error, I will change that, thanks!